.eslintrc-server-test

```diff
@@ -11,3 +11,4 @@ globals:
 rules:
   no-unused-expressions: 0  # Disable for Chai expression assertions.
   max-nested-callbacks: 0   # Disable for nested describes.
+  max-statements: [2, 20]   # More statements allowed in tests.
```

Comment:
Nope. :smile:  You're looking at the _test_ eslint, which is better with the disable because of all the mock fs / state / asserts I need for quasi-functional tests.